CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is aslways welcome, and much appreciated.
+It is also how it'll be developed.
+
+First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md).
+
+These guidelines are an attempt at better addressing the huge amount of pending
+issues and pull requests. Please read them closely.
+
+## Submitting Pull Requests
+
+### Getting started
+
+You might know the basics of 
+[contributing on GitHub](https://help.github.com/articles/using-pull-requests) and have a fork
+[properly set up](https://github.com/ohmyzsh/ohmyzsh/wiki/Contribution-Technical-Practices).
+
+You can only create PRs on a dedicated branch.
+
+If you create your own PR, please double check that everything is all right. Also please type what you fixed or added in the PR body,
+[for instance](https://help.github.com/articles/closing-issues-via-commit-messages/)
+
+## Volunteer
+
+Very nice!! :)
+
+Thank you to [all the contributers!](https://github.com/novelrt/NovelRT/graphs/contributors)
```

Comment:
```suggestion
Finally, thank you to [all of our current and future contributors!](https://github.com/novelrt/NovelRT/graphs/contributors)
```

One last suggestion here - just tweaked it a bit and corrected 1 typo (I like the premise here though)